internal/app/machined/pkg/system/health/settings.go

```diff
@@ -17,7 +17,7 @@ type Settings struct {
 
 // DefaultSettings provides some default health check settings
 var DefaultSettings = Settings{
-	InitialDelay: 200 * time.Millisecond,
-	Period:       time.Second,
+	InitialDelay: time.Second,
```

Comment:
just fyi, this also slows down boot sequence potentially, as some services wait for others to be up & healthy. not saying this change is bad, but the longer `initialDelay` is, the longer it takes for the service to become healthy